n26/cli.py

```diff
@@ -382,6 +382,15 @@ def transactions(categories: str, pending: bool, param_from: datetime or None, p
     click.echo(text.strip())
 
 
+@cli.command("make-transfer")
```

Comment:
I'd like to rename this to just `transaction`, which is more consistent with the API client and the `transactions` command.